libs/styles/styles.css

```diff
@@ -284,164 +793,305 @@
+.con-button.primary-white-button:focus {
+  background-color: var(--s2-transparent-white-900);
+}
+
+.con-button.primary-white-button:focus {
+  outline: 2px solid var(--s2-static-white-focus-indicator-color);
+}
+
+.con-button.primary-white-button[aria-disabled="true"] {
+  background-color: var(--s2-transparent-white-100);
+}
+
+/* Secondary Buttons Black */
+.con-button.secondary-black-button {
+  background-color: transparent;
+  border: 2px solid var(--s2-gray-800);
+  color: var(--s2-transparent-black-400);
+}
+
+.con-button.secondary-black-button:hover, 
+.con-button.secondary-black-button:active, 
+.con-button.secondary-black-button:focus {
+  background-color: var(--s2-gray-100);
+  border: 2px solid var(--s2-gray-900);
+}
+
+.con-button.secondary-black-button:focus {
+  outline: 2px solid var(--s2-blue-800);
+}
+
+.con-button.secondary-black-button[aria-disabled="true"] {
+  background-color: transparent;
+  outline: 2px solid var(--s2-gray-300);
+  color: var(--s2-transparent-black-400)
+}
+
+/* Secondary Buttons White */
+.con-button.secondary-white-button {
+  background-color: var(--s2-transparent-white-25);
+  border: 2px solid var(--s2-gray-800);
+  color: var(--s2-transparent-white-800)
+}
+
+.con-button.secondary-white-button:hover, 
+.con-button.secondary-white-button:active, 
+.con-button.secondary-white-button:focus {
+  background-color: var(--s2-transparent-white-100);
+  border: 2px solid var(--s2-gray-900);
+}
+
+.con-button.secondary-white-button:focus {
+  outline: 2px solid var(--s2-static-white-focus-indicator-color);
+}
+
+.con-button.secondary-white-button[aria-disabled="true"] {
+  background-color: var(--s2-transparent-white-25);
+  border: 2px solid var(--s2-transparent-white-300);
+}
+
+/* Premium Buttons  */
+.con-button.premium-button {
+  --sn-primary-button-gradient: linear-gradient(#D92361 0%, #6338EE 50%, #3B63FB 100%);
+  --sn-secondary-button-gradient: linear-gradient(#F37 0%, #8B8DFE 50%, #5681FF 100%);
+
+  font-weight: var(--s2-bold-font-weight);
+  background-color: var(--sn-primary-button-gradient);
+  border-radius: var(--s2-corner-radius-2000);
+  outline: none;
+}
+
+.con-button.premium-button:hover, 
+.con-button.premium-button:active, 
+.con-button.premium-button:focus {
+  background-color: var(--sn-secondary-button-gradient);
+}
+
+.con-button.premium-button:focus {
+  outline: 2px solid var(--s2-blue-800);
+}
+
+.con-button.premium-button[aria-disabled="true"] {
+  background-color: var(--s2-gray-100);
+  outline: none;
+}
+
+/* Disabled Button */
+.con-button[aria-disabled="true"]:hover,
+.con-button.blue[aria-disabled="true"]:hover,
+.con-button.accent-button[aria-disabled="true"]:hover,
+.con-button.fill[aria-disabled="true"]:hover,
+.con-button.primary-black-button[aria-disabled="true"]:hover,
+.con-button.primary-white-button[aria-disabled="true"]:hover,
+.con-button.secondary-black-button[aria-disabled="true"]:hover,
+.con-button.secondary-white-button[aria-disabled="true"]:hover {
```

Comment:
This seems a bit excessive 🤔 I don't see other `cursor` declarations, so would `.con-button[aria-disabled="true"]:hover` not suffice?

Reply:
Good catch, I think it would.